style.css

```diff
@@ -9,10 +9,38 @@
  * minimal changes to the HTML itself.
  */
 
+ header {
+ 	background-color: #000066;
```

Comment:
Your indentation is inconsistent. It's a minor issue but can help yourself and others when done correctly. For instance...

```css
.my-class {
  padding-top: 10px;
  padding-bottom: 10px;
  padding-left: 10px;
}
```

In sublime if we wanted to change padding to margin you can click at the start of the first padding and then hold alt and drag down to highlight a vertical selection that doesn't cover the entire line. Essentially when things are aligned we can work with them quicker and read them quicker.